src/browser/types.ts

```diff
@@ -33,6 +33,12 @@ declare global {
     namespace WebdriverIO {
         type BrowserCommand = Exclude<FunctionProperties<WebdriverIO.Browser>, EventEmitterMethod>;
         type ElementCommand = Exclude<FunctionProperties<WebdriverIO.Element>, EventEmitterMethod>;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        type OverwriteCommandFn<ThisArg, Fn extends (...args: any) => any> = (
+            this: ThisArg,
```

Comment:
Adding `this` parameter to the `OverwriteCommandFn`, so we can use `this` (usefull for elements):

```ts
browser.overwriteCommand('click', function (origCommand) {
    this // WebdriverIO.Element

    return origCommand(path);
})
```

Reply:
At the time, we can use arrow function here